argilla/docs/how_to_guides/export.md

```diff
@@ -0,0 +1,156 @@
+
+```python
+import argilla as rg
+
+client = rg.Argilla(api_url="<api_url>", api_key="<api_key>")
+
+dataset = client.datasets(name="my_dataset", workspace=workspace)
+
+dataset.to_disk(path="path/to/dataset")
+```
+
+This will save the dataset's configuration and records to the specified path. If you only want to save the dataset's configuration, you can set the `with_records` parameter to `False`.
+
+```python
+dataset.to_disk(path="path/to/dataset", with_records=False)
+```
+
+## Loading a dataset from a local disk
+
+You can load a dataset from your local disk to Argilla. This is useful if you want to restore a dataset's configuration. You can use the `rg.Dataset.from_disk` method.
+
+```python
+import argilla as rg
+
+dataset = rg.Dataset.from_disk(path="path/to/dataset")
+```
+
+# Export Records from Argilla Datasets
+
+The records alone can be exported from a dataset in Argilla.  This is useful if you want to process the records in Python, export them to a different platform, or use them in model training. All of these methods use the `rg.Dataset.records` attribute.
+
+The records can be exported as a dictionary, a list of dictionaries, or to the `datasets` package.
+
+## Export records to the `datasets` package
+
+Records can be exported from `Dataset.records` to the `datasets` package. The `to_dataset` method can be used to export records to the `datasets` package. You can specify the name of the dataset and the split to export the records.
+
+```python
+import argilla as rg
+
+client = rg.Argilla(api_url="<api_url>", api_key="<api_key>")
+
+workspace = client.workspaces("my_workspace")
+
+dataset = client.datasets(name="my_dataset", workspace=workspace)
+
+# Export records as a dictionary
+exported_ds = dataset.records.to_datasets()
+```
+
+
+## Export records to a dictionary
+
+Records can be exported from `Dataset.records` as a dictionary. The `to_dict` method can be used to export records as a dictionary. You can specify the orientation of the dictionary output. You can also decide if to flatten or not the dictionary.
+
+=== "
+
+```python
+import argilla as rg
+
+client = rg.Argilla(api_url="<api_url>", api_key="<api_key>")
+
+workspace = client.workspaces("my_workspace")
+
+dataset = client.datasets(name="my_dataset", workspace=workspace)
+
+# Export records as a dictionary
+exported_records = dataset.records.to_dict()
+# {'fields': [{'text': 'Hello'},{'text': 'World'}], suggestions': [{'label': {'value': 'positive'}}, {'label': {'value': 'negative'}}]
+
+# Export records as a dictionary with orient=index
+exported_records = dataset.records.to_dict(orient="index")
+# {"uuid": {'fields': {'text': 'Hello'}, 'suggestions': {'label': {'value': 'positive'}}}, {"uuid": {'fields': {'text': 'World'}, 'suggestions': {'label': {'value': 'negative'}}},
+
+# Export records as a dictionary with flatten=false
+exported_records = dataset.records.to_dict(flatten=True)
```

Comment:
just saw the difference between arg `orient` and `flatten`. perhaps it is orient could also be `orient="flatten"` or can the two arguments be combined?

Reply:
perhaps `to_list` complicates this?

Reply:
I'm open to making this functionality simpler/ easier to use, but I would start from an issue.